docs/examples.md

```diff
@@ -14,11 +14,26 @@ This is a good way to learn about how to call Python code from an agent.
 
 ## Music Nerd Local
 
-Exact copy of [Music Nerd](./examples/music_nerd.md) that uses an LLM that runs locally with Ollama.
+[Music Nerd Local](./examples/music_nerd_local.md) is an exact copy of
```

Comment:
Added a link to the doc of the agent network at the beginning of each sentence here and below.